README.md

```diff
@@ -129,15 +130,47 @@ cargo fmt --all
 cargo clippy --workspace --all-features -- -D warnings
 ```
 
+## Release Process
+
+### Overview
+
+We make releases within the `rust` repository specific to the crate(s) that will be consumed (e.g. agents@1.0.0, configuration@1.0.0, accumulator@1.0.0, etc).
+
+We follow [Semantic Versioning](https://semver.org/), where breaking changes constitute changes that break agent configuration compatibility.
+
+Releases are managed on GitHub [here](https://github.com/nomad-xyz/rust/releases).
+
+### Aggregating Release Notes
+
+- Want to aggregate list of all changes since last release
+- Run `git diff <sha of last release> HEAD -- **CHANGELOG.md`
+- Manually consolidate diffs into a single list for release notes
+
+### Bumping Versions
+
+- Bump package versions in all relevant `Cargo.toml` files
+- Bump the package versions in all relevant `CHANGELOG.md` files
+- E.g. for an `agents` release, this would entail bumping all agents in `rust/agents`
+- Make a PR declaring the new version you are releasing (e.g. "Bumping agents for release agents@1.0.1")
+
+### Making a New Release
+
+- Visit the [releases page](https://github.com/nomad-xyz/rust/releases) for the `rust` repo
```

Comment:
use `git tag -s` to create a signed tag
then `git push --tags` to push the tag
then use the release page